tests/ci/integration/xmlsec_patch/xmlsec_master.patch

```diff
@@ -0,0 +1,282 @@
+index b1f40619..6e53a5ef 100755
+--- a/tests/testDSig.sh
++++ b/tests/testDSig.sh
+@@ -1705,12 +1705,12 @@ execDSigTest $res_success \
+     "hmac" \
+     "--lax-key-search --hmackey certs/hmackey.bin"
+
+-execDSigTest $res_success \
+-    "phaos-xmldsig-three" \
+-    "signature-rsa-detached" \
+-    "sha1 rsa-sha1" \
+-    "rsa x509" \
+-    "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00  $url_map_rfc3161"
++# execDSigTest $res_success \
++#     "phaos-xmldsig-three" \
++#     "signature-rsa-detached" \
++#     "sha1 rsa-sha1" \
++#     "rsa x509" \
++#     "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00  $url_map_rfc3161"
+
+ execDSigTest $res_success \
+     "phaos-xmldsig-three" \
+@@ -1726,12 +1726,12 @@ execDSigTest $res_success \
+     "rsa x509" \
+     "--enabled-key-data key-value,rsa,x509 --trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00  $url_map_rfc3161"
+
+-execDSigTest $res_success \
+-    "phaos-xmldsig-three" \
+-    "signature-rsa-detached-xslt-transform-retrieval-method" \
+-    "xslt sha1 rsa-sha1" \
+-    "rsa x509" \
+-    "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00  $url_map_rfc3161"
++# execDSigTest $res_success \
++#     "phaos-xmldsig-three" \
++#     "signature-rsa-detached-xslt-transform-retrieval-method" \
++#     "xslt sha1 rsa-sha1" \
++#     "rsa x509" \
++#     "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00  $url_map_rfc3161"
+
+ execDSigTest $res_success \
+     "phaos-xmldsig-three" \
+@@ -1741,54 +1741,54 @@ execDSigTest $res_success \
+     "--enabled-key-data key-value,rsa,x509 --trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00  $url_map_rfc3161"
+
+
+-execDSigTest $res_success \
+-    "phaos-xmldsig-three" \
+-    "signature-rsa-enveloped" \
+-    "enveloped-signature sha1 rsa-sha1" \
+-    "rsa x509" \
+-    "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00"
++# execDSigTest $res_success \
++#     "phaos-xmldsig-three" \
++#     "signature-rsa-enveloped" \
++#     "enveloped-signature sha1 rsa-sha1" \
++#     "rsa x509" \
++#     "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00"
+
+-execDSigTest $res_success \
+-    "phaos-xmldsig-three" \
+-    "signature-rsa-enveloping" \
+-    "sha1 rsa-sha1" \
+-    "rsa x509" \
+-    "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00"
++# execDSigTest $res_success \
++#     "phaos-xmldsig-three" \
++#     "signature-rsa-enveloping" \
++#     "sha1 rsa-sha1" \
++#     "rsa x509" \
++#     "--trusted-$cert_format certs/rsa-ca-cert.$cert_format --verification-gmt-time 2009-01-01+10:00:00"
```

Comment:
Your PR description indicated that the patch commented out tests related to MD5?  But these relate to rsa-sha1?